content/articles/getting-started-with-css/index.md

```diff
@@ -0,0 +1,194 @@
+```html
+<style>
+ #Properties-of-CSS{
+    background-color: black;
+ }
+</style>
+```
+### Using Hex Codes
+Hex is the short name for Hexadecimal code. Hex code uses six hexadecimal numbers to specify colors in CSS. Some Hex values of different colors according to [Simmons](http://web.simmons.edu/) are;
+- black  #000000
+
+- maroon  #800000
+
+- yellow  #FFFF00
+
+- purple  #800080
+
+Representing color in Hex code will be like;
+```html
+<style>
+   p{
+   color: #800080;
+   }
+</style>
+```
+ But since it is hard to remember hex code,one can shorten it. This are examples from [freshersnow](https://tutorials.freshersnow.com/css/css-colors/);
+ - black  #000
+
+ - red  #F00
+
+ - cyan  #0FF
+
+### Using RGB colors
+RGB stands for red, green and blue. It is another way of representing colors in CSS.Each value could have a value from between o to 100% or 0 to 255. A value of 0 means none of the colors was used while a value of 255 or 100% means all of that color was used. The color black would result from all values (red, green and blue)being 0 while the color white would result from all the values being a 100%. Some of RGB values according to [rapidtables](https://www.rapidtables.com/web/color/RGB_Color.html) are as follows;
+- black  rgb(0, 0, 0)
+
+- red  rgb(255, 0, 0)
+
+- grey  rgb(128, 128, 128)
+
+RGB values can be styled in CSS as follows;
+```html
+<style>
+   p{
+   color: rgb(128, 128, 128);
+   }
+</style>
+```
+Let us use RGB values to represent an Id attribute;
+```html
+<style>
+   #Properties-of-CSS{
+                color: rgb(128, 128, 128);
+                }
+</style>
+```
+
+
+
+## 3.Changing the Font Size and Font Family of an Element
+In order to change the font size of an element,the **font-size** element is introduced like this;
+```html
+h2{
+    font-size: 15px;
+}
+```
+Say for example one wants to change the font-size of a paragraph to 17px within the style element it will be like;
+
+ ```html
+p{
+    font-size: 17px;
+}
+</style>
+```
+
+In order to change the font name of an element,the **font-name** element is introduced like this;
+```html
+<style>
+ p{
+     font-family: Helvetica;
+ }
+</style>
+```
+## 4.Sizing Images
+CSS uses a property known as **width** to size elements by the help of class declarations such as **.smaller-width** and **.larger-image** like this;
+```html
+<style>
+ .smaller-image{
+     width: 120px;
+ }
+ </style>
+```
+This should also be incorporated into your HTML element like this;
+```html
+<img class= "smaller-image">
+```
+
+## 5.Styling HTML Using CSS
+  Let us create a HTML element and style it using CSS;
+  ```html
+  <!DOCTYPE html>
+  <html>
+  <h1>Getting Started With CSS</h1>
+  <style>
+   body{
+       color: black;
+       background-color: blue;
+       font-family: monospace;
+       font-size: 16px;
+   }
+   </style>
+   </html>
+   ```
```

Comment:
This section should tie together everything you've shown the reader. It does not do that. Consider revisiting the section and adding content that would tie together the article in a manner that adds value to it.